data-prepper-core/src/main/java/org/opensearch/dataprepper/sourcecoordination/LeaseBasedSourceCoordinator.java

```diff
@@ -322,7 +321,12 @@ public void updatePartitionForAcknowledgmentWait(final String partitionKey, fina
 
         itemToUpdate.setPartitionOwnershipTimeout(Instant.now().plus(ackowledgmentTimeout));
 
-        sourceCoordinationStore.tryUpdateSourcePartitionItem(itemToUpdate);
+        try {
+            sourceCoordinationStore.tryUpdateSourcePartitionItem(itemToUpdate);
+        } catch (final Exception e) {
+            LOG.info("Exception while updating acknowledgment wait for the partition {}: {}", partitionKey, e.getMessage());
```

Comment:
This should be an `ERROR` log and should include the stack trace since you don't know the underlying cause if you reach this code.

Reply:
Will update